### PR TITLE
Add lv.martinsz.millionaire

### PR DESCRIPTION
**Pull request by martinszeltins**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by hfiguiere** [comment]
closing PR make review more difficult. You don't need to do it to update it.

**Comment by martinszeltins** [comment]
> closing PR make review more difficult. You don't need to do it to update it.

Sorry, I didn't know that.

File: lv.martinsz.millionaire.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
```
Comment (TingPing) [inline lv.martinsz.millionaire.json:23]: These seem to be a random copy paste list. None of them apply to your project as-is.

File: lv.martinsz.millionaire.json [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    ],
+    "modules" : [
+        {
+            "name" : "blueprint",
```
Comment (TingPing) [inline lv.martinsz.millionaire.json:27]: Missing `cleanup` for this project.

File: lv.martinsz.millionaire.json [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+                {
+                    "type" : "git",
+                    "url" : "https://gitlab.gnome.org/jwestman/blueprint-compiler.git",
+                    "tag" : "v0.4.0"
+                }
+            ]
+        },
+        {
+            "name" : "millionaire",
+            "builddir" : true,
```
Comment (TingPing) [inline lv.martinsz.millionaire.json:39]: You can remove this. It's always true for `meson`.

**Comment by TingPing** [comment]
bot, build lv.martinsz.millionaire

**Comment by TingPing** [comment]
Builds fine your appdata file just fails `appstream-util validate`.

**Comment by TingPing** [comment]
I would remove this entire section:

```xml
      <p>Created using these technologies</p>
      <ul>
        <li>GTK 4</li>
        <li>Libadwaita</li>
        <li>C</li>
      </ul>
```

The description is for a general audience and not developers.

**Comment by martinszeltins** [comment]
Pushed a new commit and fixed all the issues [[8b00753] Updated to v1.0.2 - fixes](https://github.com/flathub/flathub/pull/3658/commits/8b0075324c6e77e1c8e733bbcadf1182a38ee40c)

**Comment by martinszeltins** [comment]
bot, build lv.martinsz.millionaire

**Pull request closed without merging**